core/src/i18n/pl.yaml

```
@@ -128,6 +128,25 @@ settings:
   confirmations: Potwierdzenia
   dontConfirmDelete: Usuwanie bez potwierdzenia
   language: Język
+common:
+  name: Nazwa
+  created: Stworzono
+  labels: Etykiety
+  status: Status
+  buttons:
+    delete: Usuń
+    cancel: Anuluj
+    edit-yaml: Edituj YAML
+  delete-dialog:
+    delete-confirm: Nie pytaj ponownie
+    message: Czy na pewno chcesz usunąć {{type}} {{name}}?
+    title: Usuń {{name}}
+  tooltips:
+    copy-button: Skopiuj do schowka
+    duplicate-key: Zduplikowany klucz
+    k8s-name-input: Nazwa musi zawierać małe znaki alfanumeryczne, może zawierać „-” (np. „my-name1”).
```

If I understand correctly, what you actually mean here is that the name must **consist of** lowercase alphanumeric characters and "-". 
`Contain` suggests that you require that at least one of them is in the name, just like in the requirements for passwords.
```suggestion
    k8s-name-input: Nazwa musi składać się z małych znaków alfanumerycznych, może zawierać również „-” (np. „my-name1”).
```
or
`z małych liter i cyfr` instead of `małych znaków alfanumerycznych`